tap_stripe/__init__.py

```diff
@@ -910,7 +964,6 @@ def main():
         Context.config = args.config
         Context.state = args.state
         configure_stripe_client()
-        validate_dependencies()
```

Comment:
Remove defined function `validate_dependencies()` from the code if it is not used anywhere.